src/content/SubmissionModeration/Rejection.tsx

```diff
@@ -32,15 +32,13 @@ const createRejectionComment = (ruleLetters: string[]) => {
   // TODO: externalize this - wiki?
   return `Hello there. I'm a mod in /r/NeutralPolitics.
 
-We appreciate your participation in the sub, but we did not approve this submission, because it doesn't conform to our [submission rules.](http://www.reddit.com/r/NeutralPolitics/wiki/guidelines#wiki_submission_rules)
+We appreciate your participation in the subreddit, but we did not approve this submission, because it doesn't conform to 
+our [submission rule(s)](http://www.reddit.com/r/NeutralPolitics/wiki/guidelines#wiki_submission_rules) Specifically,
+rule${multipleRules ? 's' : ''} ${formattedRuleLetters}.
```

Comment:
looking at what nose posted, i think he wants:

```suggestion
We appreciate your participation in the subreddit, but we did not approve this submission, because it doesn't conform to 
our [submission rule(s)](http://www.reddit.com/r/NeutralPolitics/wiki/guidelines#wiki_submission_rules) ${formattedRuleLetters}.
```

which would read like "doesn't conform to our submission rule(s) A, B and G."

i'm also not sure how reddit will render this with the line breaks - does it turn them into spaces or? previously there were no line breaks for this sentence.